Objective-C/Tests/QueryTest.m

```diff
@@ -922,6 +922,30 @@ - (void) testParameters {
     AssertEqual(numRows, 4u);
 }
 
+- (void) testStringQueryParameter {
```

Comment:
It's it possible to rewrite and merge testParameters and testStringQueryParameter?

Reply:
Re-wrote the function and mixed to a single function for checking string, number, integer, long long, float, double, boolean, date. Remaining are the Blob, Dictionary and Array which we will add a separate function.